rosidl_typesupport_c/package.xml

```diff
@@ -4,8 +4,10 @@
   <name>rosidl_typesupport_c</name>
   <version>1.0.0</version>
   <description>Generate the type support for C messages.</description>
-  <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
+  <maintainer email="clalancette@openrobotics.org">Chris Lalancette</maintainer>
+  <maintainer email="sloretz@openrobotics.org">Share Loretz</maintainer>
```

Comment:
Share -> Shane :)

Reply:
Apparently I can't type :).  Fixed in dfc827f.